api/comments/serializers.py

```diff
@@ -111,6 +113,8 @@ def update(self, comment, validated_data):
                     comment.edit(content, auth=auth, save=True)
                 except PermissionsError:
                     raise PermissionDenied('Not authorized to edit this comment.')
+                except ValidationValueError:
+                    raise ValidationError('Ensure this field has no more than 500 characters.')
```

Comment:
`ValidationValueError` is a [subclass](https://github.com/CenterForOpenScience/modular-odm/blob/d13f941e4d6c90a4840c492b7225e8e85dafeea7/modularodm/exceptions.py#L41-L44) of `ValidationError`. Why do we need to capture and reraise here at all? Why not let the `ValidationValueError` itself raise?